geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert2/transforms/DateFunctionFactory.scala

```diff
@@ -120,8 +141,6 @@ class DateFunctionFactory extends TransformerFunctionFactory {
             .withZone(ZoneOffset.UTC)
     }
 
-  // TODO add more default dates for type inference
```

Comment:
Did you Pokemon this and get 'em all?

Reply:
yeah, I added all the predefined ones in the java DateTimeFormatter class